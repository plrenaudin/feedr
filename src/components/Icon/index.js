const Icon = props => (
  <span class={`icon-wrapper${(props.class && " " + props.class) || ""}`}>
    <svg class="icon">
      <use xlinkHref={`#icon-${props.name}`} />
    </svg>
  </span>
);

export default Icon;
