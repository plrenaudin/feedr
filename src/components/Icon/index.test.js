import { render } from "inferno";
import Icon from ".";

it("Matches the snapshot", () => {
  const div = document.createElement("div");
  expect(render(<Icon name="price-tag" />, div).dom).toMatchSnapshot();
});
