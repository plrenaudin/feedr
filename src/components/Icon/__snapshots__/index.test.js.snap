// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<span
  class="icon-wrapper"
>
  <svg
    class="icon"
  >
    <use
      xlink:href="#icon-price-tag"
    />
  </svg>
</span>
`;
